app/views/common_parameters/_puppetclasses_parameters.html.erb

```diff
@@ -4,8 +4,8 @@
     <tr>
       <th class='col-md-3'><%= _("Puppet class overrides") %></th>
       <th class='col-md-2'><%= _("Name") %></th>
-      <th class='col-md-5'><%= _("Value") %></th>
-      <th class='col-md-3'><%= _("Use Puppet default") %></th>
+      <th class='col-md-4'><%= _("Value") %></th>
+      <th class='col-md-2'><%= _("Use Puppet default") %></th>
       <th><%= _('Actions') %></th>
```

Comment:
col-md-1 for Actions then?

One of the issues I'm seeing is that the rendered table has different col-md- values than the titles, which makes these col-md values almost useless. Perhaps imitating the boostrap example would be easier.